CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [Version 1.3.0](https://github.com/dataiku/dss-plugin-pi-server/releases/tag/v1.3.0) - Feature and bug release - 2025-05-23
+
+- Accept timezone in datetime inputs
+- Add a "Step" column
+- Add linear interpolation to Transpose & Sync recipe
+- Allows maxCount parameter tuning to allow faster data transfer
```

Comment:
We should probably mention that it is not available for event frames stuff? Say something like:

`(not available for Event frames related components - for these batch mode may be useful to speed up data transfer)`

Reply:
done in [2db7457](https://github.com/dataiku/dss-plugin-osisoft/pull/74/commits/2db7457adda2a4991f82acea63641451b6d6b40b)